lib/oli/accounts/accounts.ex

```diff
@@ -1,116 +1,34 @@
 defmodule Oli.Accounts do
-  @moduledoc """
```

Comment:
Why would we removing all the documentation here? It's very convenient when hovering over the functions for quick reference in other parts of the codebase

Reply:
Personally, I feel that the generated docs just make the file harder to read. they all use the same conventions, so the docs don't really tell you anything.